scripts/common_click.inc

```diff
@@ -252,11 +252,12 @@ function drawWater()
     local water = srFindImage("iconWaterJugSmall.png", 1);
     if water then
       safeClick(water[0]+3, water[1]-5);
+	  lsSleep(click_delay*5);
       local max = waitForImage("crem-max.png", 500, "Waiting for Max button");

```

Comment:
Doesn't waitForImage already sleep an unlimited amount of time until the image shows up?  Or is it only sleeping 500ms and would it just be better to increase the timeout passed to that function.

Also, indentation seems to have gotten a little messed up here (wasn't great to start with either, it looks like ^_^).